sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 ### Breaking Changes
+ - [Python 3.7 reached end-of-life on June 27th 2023](https://devguide.python.org/versions/). Consequently, 3.7 will be deprecated in azure-ai-ml starting in October 2023 and azure-ai-ml will end support for 3.7 in February 2024.
```

Comment:
Curious if you could elaborate on why this is a changelog item for a specific release? (I'm just curious)

Reply:
1. Paul wants this message out ASAP and 1.11.0 is the next possible window
2. Azure guidelines state that we have to announce these changes to users and can only announce these sorts of deprecations/end of support in October and February, so this will make sure that the news is propagated for the next release which should land at the start of October